docs/using/backup.md

```diff
@@ -57,46 +46,75 @@
 
     - *Name*:
     - *Description*:
-    - *Type:*
-        - *S3*:
+    - *Type*: Choose the type of storage.
+        - *S3*: Use [Amazon AWS S3]
+            - *Endpoint*: The S3 backup location endpoint.
+            - *Bucket Name*:
+            - *Access Key*:
+            - *Secret Key*:
         - *Local Client:* (Not currently implemented)
         - *Local Server:* (Not currently implemented)
-    - *Endpoint:*
-    - *Bucket Name:*
-    - *Access Key:*
-    - *Secret Key:*
 
 4. Click *Add* to add the location or *Test* to test the connection.
 
-## Creating a backup
+## MySQL backup preconditions
 
-!!! hint alert alert-success "Tip"
-    You must [add a backup storage location](#adding-a-storage-location) before backing up.
+- There is only one MySQL instance running on the node.
 
-1. Navigate to *Backup Management* → *Backup Inventory*.
+- MySQL is running:
+
+    - as a service via `systemd`;
+
+    - with the name `mysql` (to confirm, use `systemctl status mysql`);
+
+    - from a `mysql` system user account.
+
+- There is a `mysql` system group.
+
+- MySQL is using the `/var/lib/mysql` directory for database storage.
```

Comment:
Also `pmm-agent` should have write permissions to this dir. 

@oter am I right?

Reply:
yes, you are right

Reply:
```suggestion
- MySQL is using the `/var/lib/mysql` directory for database storage.

- `pmm-agent` has read/write permissions to the `/var/lib/mysql` directory.
```